### ISSUE TITLE
Cannot tab to Mergetag Selector

### ISSUE THREAD

**Issue by jakejackson1** [GravityPDF/gravity-pdf#1226]
**Description**
On the PDF creation page, when tabbing through the UI I am unable to tab directly to the Mergetag selector and navigate through the component using my keyboard only. 

**Step To Reproduce**
1. Go to PDF List page and add a new PDF
2. Fill in the Name setting then tab through the components until you get to the Filename
3. When you next tab you should go to the Mergetag selector, but do not.

**Expected behavior**
To be able to navigate and interact with the UI via the keyboard alone.

**Screenshots**
<img width="874" alt="Screen Shot 2021-03-14 at 8 24 31 pm" src="https://user-images.githubusercontent.com/2918419/111065079-85232080-8503-11eb-84e4-09143dc0b1e2.png">
